database/mysql/README.md

```diff
@@ -1,55 +0,0 @@
-# MySQL
-
-`mysql://user:password@tcp(host:port)/dbname?query`
```

Comment:
Any thoughts we might end up using this for migrations of flagship?

Reply:
We would never :-) We have several other well used migration tools